rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -852,13 +845,13 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
           case READY:
             return childPolicyWrapper.getPicker().pickSubchannel(rlsAppliedArgs);
           case TRANSIENT_FAILURE:
-            return handleError(rlsAppliedArgs, Status.INTERNAL);
+            // fall-through
```

Comment:
nit: No need comment. The code explains itself.

Reply:
done.